Shared.Tests/IntegrationTestBase.cs

```diff
@@ -69,6 +69,8 @@ public partial class IntegrationTestBase
         protected string m_apiTag;
 
         protected const string m_simpleTransformationAsString = "c_scale,w_0.5";
+        protected const string m_simpleTransformationWithExtAsString = "c_scale,w_0.5/jpg";
```

Comment:
Do you really need both m_simpleTransformationWithExtAsString and m_updateTransformationWithExtAsString?